test/ray_serializer.jl

```diff
@@ -77,3 +77,25 @@ end
         @test deserialize(s) == x
     end
 end
+
+@testset "serialize_to_ray_object" begin
+    @testset "nested objects" begin
+        obj1 = Ray.put(1)
+        obj2 = Ray.put(2)
+
+        objs = [obj1, obj2]
+        ray_obj = Ray.serialize_to_ray_object(objs)
+        nested_obj_ids = ray_jll.GetNestedRefIds(ray_obj[])
+
+        @test issetequal([o.oid for o in objs], nested_obj_ids)
+        @test Ray.get(ray_obj) == objs
+
+        stuff2 = [obj1, (obj2, obj1, "blah"), 1]
+        ray_obj2 = Ray.serialize_to_ray_object(stuff2)
+        nested_obj_ids2 = ray_jll.GetNestedRefIds(ray_obj2[])
+        @test length(nested_obj_ids2) == 2
+        @test issetequal(nested_obj_ids2, nested_obj_ids)
+
+        @test Ray.get(ray_obj2) == stuff2
+    end
+end
```

Comment:
Seems like more of an `object_store.jl` test. Is there a way to test this without calling `get`/`put` here?

Reply:
in this particular case I think we could get away with random ones.  but once we start trying to register ownership, I think things will blow up (need to be a "real" object ref, not a random one for that)